package.json

```diff
@@ -42,6 +43,7 @@
     "@rollup/plugin-commonjs": "^22.0.2",
     "@rollup/plugin-json": "^4.1.0",
     "@rollup/plugin-node-resolve": "^14.0.0",
+    "@types/jest": "^29.4.0",
```

Comment:
there were two option here `@types/jest` and `@jest/globals`. But auto import for test functions didn't work with `@jest/globals` so I used `@types`

https://jestjs.io/docs/getting-started#type-definitions